src/main.d.ts

```diff
@@ -22,17 +35,16 @@ export interface MajorNodeVersions {
   lts?: string
 }
 
-export interface NodeVersions {
+export interface NodeVersion {
```

Comment:
Good point about making `MajorNodeVersion` singular. However, `NodeVersions` might need to be remain plural, since this contains several releases.

Reply:
Well, I usually follow this pattern:
`type MyObject = {...}` and then `type MyObjects = MyObject[]`
Because even though we have versions in `NodeVersions`, the `NodeVersions` itself is a pretty singular instance, it's not an array... But I guess it's a matter of preference, I'd be ok with `NodeVersions`.
That being said, I renamed it to `AllNodeVersions` to match the package name, since this interface represents the default export of the package, felt like this might be the best fitting name. Thoughts?

Reply:
Yes, `AllNodeVersions` is even better :+1: